feature-pack/src/main/resources/modules/system/layers/base/org/jboss/jts/main/module.xml

```diff
@@ -47,5 +47,11 @@
         <module name="org.jboss.weld.core"/>
         <module name="javax.annotation.api" export="true" />
         <module name="javax.interceptor.api" export="true" />
+        <module name="org.wildfly.transaction.client"/>
+        <module name="org.jboss.as.transactions">
+            <imports>
+                <include path="org.jboss.as.txn.integration"/>
```

Comment:
@dmlloyd from my experiments both `/` and `.` work fine. Is the `/` preferred way to use? What I can see (and where I took inspiration) all transaction related module.xml(s) use `.`.